lib/galaxy/datatypes/media.py

```diff
@@ -237,10 +310,7 @@ class Mp3(Audio):
     file_ext = "mp3"
 
     def sniff(self, filename: str) -> bool:
-        if which("ffprobe"):
-            metadata, streams = ffprobe(filename)
-            return "mp3" in metadata["format_name"].split(",")
-        return False
+        return _get_file_format_from_magic_number(filename, "mp3")
```

Comment:
but yes cases like this, instead of `return False`, just `return _get_file_format_from_magic_number(filename, "mp3")` and keeping the ffprobe would potentially be good.